airflow/contrib/operators/kubernetes_pod_operator.py

```diff
@@ -171,6 +171,69 @@ def execute(self, context):
         except AirflowException as ex:
             raise AirflowException('Pod Launching failed: {error}'.format(error=ex))
 
+    def create_new_pod_for_operator(self, labels, launcher):
+        """
+        Create a new pod and monitor it to completion
+        :param labels:
```

Comment:
```suggestion

                :param labels:
```
Otherwise, docs doesn't looks good.
https://github.com/apache/airflow/pull/6344#discussion_r336709991